pegasus/sites.v3/code.org/public/css/tools.css

```diff
@@ -58,14 +58,83 @@ h3.tutorial-info-h {
   margin-top: 0px;
 }
 
+b.purple {
+  font-size: 14px;
+  color: #7665a0;
+}
+
 button.tutorial-gray {
   background-color: #e7e8ea;
   border-color: #e7e8ea;
   color: #5b6770;
 }
 
+.pl-callout {
+  border: 1px solid #0094ca;
+  height: 72px;
+  margin: 20px 0px;
+}
+
+.pl-icon {
+  background-color: #0094ca;
+  width: 72px;
+  height: 70px;
+  float: left;
+  padding: 16px 16px;
+}
+
+.pl-book {
+  color: rgba(255, 255, 255, 0.8);
+  font-size: 38px;
+}
+
+.pl-description {
+  float: left;
+  padding: 14px 15px;
+  width: 860px;
```

Comment:
Does this need to be responsive? I assume so since it's on Pegasus. If so, I don't think we can hard-code a width this big.

Reply:
It's responsive, scroll down in this file for mobile css.

Reply:
Cool. Never mind, then! 🙂